### ISSUE TITLE
Deep Ritz method

### ISSUE THREAD

**Issue by ChrisRackauckas** [SciML/NeuralPDE.jl#201]
https://arxiv.org/pdf/1710.00211.pdf

**Comment by khizirsiddiqui** [comment]
Hey @ChrisRackauckas , I would like to work on this. Commenting just to confirm no one else is already here. :)

**Comment by ChrisRackauckas** [comment]
Yes, no one that I know of is working on this. It's hard to know if it can be applied generally though: Deep Galerkin might be a better starting point.

**Comment by ChrisRackauckas** [comment]
I got permission to share this as well:

[18_337_Final_Project_Report.pdf](https://github.com/SciML/NeuralPDE.jl/files/5765389/18_337_Final_Project_Report.pdf)

[Uploading DeepProject.zip…]()

**Comment by khizirsiddiqui** [comment]
Sorry, got a little busy somewhere. Whoaa! This is such a nice report, thanks for this! I will start with Deep Galerkin and keep you updated.
